package/MDAnalysis/analysis/waterdynamics.py

```diff
@@ -820,7 +825,8 @@ def _selection_serial(self, universe, selection_str):
         selection = []
         for ts in universe.trajectory:
             selection.append(universe.select_atoms(selection_str))
-            print(ts.frame)
+            sys.stdout.write("\rSelecting atoms from frame: %i"%ts.frame)
```

Comment:
There should be a verbosity argument to the function. I honestly never look at the log and I'm not sure how many people know about it.